src/interfaces/DebugDiscoveryParams.ts

```diff
@@ -0,0 +1,10 @@
+export interface DebugDiscoveryParms {
```

Comment:
Och no :( I never noticed the typo.  Well we can use that for 0.4.2 then :S

Reply:
Whoops. We can also just fix and and do a quick release. This one is super quick, but if not no worries.

Reply:
Haha, I think this is fine for now :P